modules/tensor_mechanics/test/tests/pressure/pressure_aux.i

```diff
@@ -0,0 +1,159 @@
+#
+# Pressure Test
+#
+# This test is designed to compute pressure loads on three faces of a unit cube.
+# The pressure is computed as an auxiliary variable. It should give the same result
+# as pressure_test.i
+#
+# The mesh is composed of one block with a single element.  Symmetry bcs are
+# applied to the faces opposite the pressures.  Poisson's ratio is zero,
+# which makes it trivial to check displacements.
+#
+
+
+[Mesh]
+  type = FileMesh
+  file = pressure_test.e
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[Functions]
+  [./rampConstant]
+    type = PiecewiseLinear
+    x = '0. 1. 2.'
+    y = '0. 1. 1.'
+    scale_factor = 1.0
+  [../]
+  [./zeroRamp]
+    type = PiecewiseLinear
+    x = '0. 1. 2.'
+    y = '0. 0. 1.'
+    scale_factor = 2.0
+  [../]
+  [./rampUnramp]
+    type = PiecewiseLinear
+    x = '0. 1. 2.'
+    y = '0. 1. 0.'
+    scale_factor = 10.0
+  [../]
+[]
+
+[Variables]
+  [./disp_x]
+  [../]
+  [./disp_y]
+  [../]
+  [./disp_z]
+  [../]
+[]
+
+[AuxVariables]
+  [./pressure_1]
+  [../]
+  [./pressure_2]
+  [../]
+  [./pressure_3]
+  [../]
+[]
+
+[AuxKernels]
+  [./side1_pressure_ak]
+    type = FunctionAux
+    variable = pressure_1
+    boundary = 1
+    function = rampConstant
+  [../]
+  [./side2_pressure_ak]
+    type = FunctionAux
+    variable = pressure_2
+    boundary = 2
+    function = zeroRamp
+  [../]
+  [./side3_pressure_ak]
+    type = FunctionAux
+    variable = pressure_3
+    boundary = 3
+    function = rampUnramp
+  [../]
+[]
+
+[Kernels]
+  [./TensorMechanics]
+    displacements = 'disp_x disp_y disp_z'
+  [../]
+[]
```

Comment:
Use the master action here too, and remove the variables, kernel, and strain block